python/code_to_gast/py_loop.py

```diff
@@ -41,9 +41,14 @@ def for_range_statement_to_gast(node):
         gast["type"] = "forRangeStatement"
         ast.dump(node)
         args = node.iter.args
-        step_num = get_step_num(args)
```

Comment:
per our live conversation, it makes a lot more sense to use the helper function pattern to handle only one argument rather than adding in a conditional to this level. Basically just add an if statement into `get_step_num` and then create `get_start_num` and `get_end_num` functions that do effectively the same thing. I think it will make it much clearer over time